app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,4 @@
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
```

Comment:
we could probably apply this globally via a lint configuration in gradle but it's not really necessary now

Reply:
no it was to suppress the ide error

Reply:
no gradle shouldn't because it's an android linter configuration. I just meant that it could be configured via gradle so that it applies to other language's xml files too

Reply:
see https://github.com/X1nto/Mauth/blob/master/app/build.gradle.kts#L68-L71 for example